jobs/JEVS_GLOBAL_ENS_PLOTS

```diff
@@ -1,79 +1,130 @@
-#!/bin/ksh
-
-########################################
-# Preliminary data setup step
-########################################
+#!/bin/bash
+#  JOB card created for running METPlus-based Global enesmble verifications
+#   By Binbin Zhou, IMSG@EMC/NCEP
+#   Feb 2, 2022
+#
+#   August 14 2024     Ho-Chun Huang    Merged with JEVS_GLOBAL_ENS_CHEM_GRID2OBS_PLOTS
```

Comment:
@Ho-ChunHuang-NOAA Can this line, which is now outdated, be changed from JEVS_GLOBAL_ENS_CHEM_GRID2OBS_PLOTS to JEVS_GLOBAL_ENS_PLOTS?

@malloryprow

Reply:
I am using JEVS_GLOBAL_ENS_PLOTS as the base to include global_ens_chem part.  Want to confirm you want me to do the changes as specified.